app/models/evmbaseline.rb

```diff
@@ -1,31 +1,49 @@
+# baseline model
 class Evmbaseline < ActiveRecord::Base
   unloadable
 
-  attr_protected :id
-
-  has_many :evmbaselineIssues, dependent: :delete_all
+  # Relations
   belongs_to :author, class_name: 'User'
   belongs_to :project
+  has_many :evmbaselineIssues, dependent: :delete_all
 
+  # attribute
+  attr_protected :id
+
+  # Validate
   validates :subject, presence: true
 
+  # Minimum start date of baseline.
+  #
+  # @return [date] minimum of start date
   def minimum_start_date
     evmbaselineIssues.minimum(:start_date)
   end
 
+  # maximum due date of baseline.
+  #
+  # @return [date] maximum of due date
   def maximum_due_date
     evmbaselineIssues.maximum(:due_date)
   end
 
+  # BAC of baseline.
+  #
+  # @return [numeric] BAC of baseline
   def bac
     evmbaselineIssues.sum(:estimated_hours).round(1)
   end
 
+  acts_as_searchable columns: ["#{table_name}.subject", "#{table_name}.description"],
```

Comment:
__[rubocop]__
* Line is too long. [85/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)